src/test/java/LoginTests.java

```diff
@@ -1,25 +1,55 @@
+import Pages.HomePage;
+import Pages.LoginPage;
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.chrome.ChromeDriver;
 import org.openqa.selenium.chrome.ChromeOptions;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 import org.testng.Assert;
 import org.testng.annotations.Test;
 
 import java.time.Duration;
 
 public class LoginTests extends BaseTest {
+    private static final Logger log = LoggerFactory.getLogger(LoginTests.class);
+    // @Test
+
+
     @Test
-    public void loginEmptyEmailPassword() {
+    public void validLogin() {
+        LoginPage loginPage = new LoginPage(driver);
+        HomePage homePage = new HomePage(driver);
 
-//      Added ChromeOptions argument below to fix websocket error
-        ChromeOptions options = new ChromeOptions();
-        options.addArguments("--remote-allow-origins=*");
+        loginPage.login();
+        Assert.assertTrue(homePage.getUserAvatar().isDisplayed());
+    }
 
-        WebDriver driver = new ChromeDriver(options);
-        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
+   @Test
+    public void loginValidEmailInvalidPasswordTest() {
+        LoginPage loginPage = new LoginPage(driver);
+        loginPage.loginValidEmailInvalidPassword();
+        String url = "https://qa.koel.app/";
+        Assert.assertEquals(driver.getCurrentUrl(), url);
+
+    }
 
+    @Test
+    public void loginInvalidEmailValidPasswordTest() {
+
+        LoginPage loginPage = new LoginPage(driver);
+        loginPage.loginInvalidEmailValidPassword();
         String url = "https://qa.koel.app/";
-        driver.get(url);
         Assert.assertEquals(driver.getCurrentUrl(), url);
-        driver.quit();
     }
-}
+
+    @Test
+    public void loginEmptyEmailPassword() {
+        LoginPage loginPage = new LoginPage(driver);
+
+
+        loginPage.provideEmail("");
+        loginPage.providePassword("");
+     loginPage.clickSubmit();
+        Assert.assertEquals(driver.getCurrentUrl(), url);
```

Comment:
How can I beautify code in IntelliJ:

The formatting shortcuts in Intellij IDEA are :
For Windows : Ctrl + Alt + L.
For Ubuntu : Ctrl + Alt + Windows + L.
For Mac : ⌥ (Option) + ⌘ (Command) + L.

![image](https://user-images.githubusercontent.com/104761482/209774134-9e63bfed-d768-4455-a860-028305c85e17.png)